apps/condo/domains/common/components/Comments/Comment.tsx

```diff
@@ -191,7 +196,30 @@ export const Comment: React.FC<ICommentProps> = ({ comment, updateAction, delete
                 </Typography.Text>
             }
             author={comment.user.name}
-            datetime={formatDate(intl, comment.createdAt)}
+            datetime={
+                <div
+                    onMouseOut={() => setDateShowMode('created')}
+                    onMouseOver={() => setDateShowMode('updated')}
+                >
+                    <span
+                        style={{
+                            opacity: dateShowMode === 'created' ? 1 : 0,
```

Comment:
Try to use `visibility` css property, because we will not tied to opacity prop, we might don't know in advance.

Reply:
@AntonAL  Opacity is used here for smooth transition

Reply:
It blinks and looks a little bit annoying. I think, we don't need the animation.